system_monitoring_system/gui.py

```diff
@@ -667,6 +675,7 @@ def main():
 
     while True:
         event, values = window.read()
+        print(event, values)
```

Comment:
Can you remove this print statement? And if there are any other print statements then that too

Reply:
The print statement is removed in #34, so squash and merge of that will remove it automatically.

Reply:
Ok